media-libs/quesoglc/quesoglc-0.7.2-r1.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Free implementation of the OpenGL Character Renderer (GLC)"
+HOMEPAGE="http://quesoglc.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}-free.tar.bz2"
+
+LICENSE="LGPL-2.1+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="doc examples static-libs"
+
+RDEPEND="dev-libs/fribidi
+	media-libs/fontconfig
+	media-libs/freetype:2
+	virtual/glu
+	virtual/opengl"
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig
+	doc? ( app-doc/doxygen )"
```

Comment:
I had to add `dev-texlive/texlive-latexextra` as well as otherwise the build would produce an error:
`! LaTeX Error: File newunicodechar.sty' not found.` (interestingly the build didn't fail actually, luckily i saw it in the output)
newunicodechar is being installed with `dev-texlive/texlive-latexextra`